hapic/ext/bottle/context.py

```diff
@@ -64,6 +65,15 @@ def get_request_parameters(self, *args, **kwargs) -> RequestParameters:
             files_parameters=files_parameters,
         )
 
+    def get_file_response(
+        self, file_response: HapicFile, http_code: int
+    ) -> bottle.HTTPResponse:
+        if file_response.file_path:
```

Comment:
Don't forgot to test all case of ̀HapicFile` before MR opening.

Reply:
Done.